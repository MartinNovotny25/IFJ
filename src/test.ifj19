ahoj = 1
dublik = 21.3
beta = 'betka je stetka'
