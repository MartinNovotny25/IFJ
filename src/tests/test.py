def foo():
    x = 1
    return x
def main():
    # assigment
    x, y, z = 'foo', 14, 21.1
    print(x, y, z)

    # for i in range(start, stop, step)
    for i in range(1, 6):
        print(i)

    #

    # function call
    y = foo()

if __name__ == '__main__':
    # main
    main()
