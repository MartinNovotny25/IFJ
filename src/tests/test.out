TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, -1619581712)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(-)
TOKEN(IDENTIFIKATOR)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(/)
TOKEN(IDENTIFIKATOR)
TOKEN(//)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, -1619581136)
TOKEN(>)
TOKEN(()
TOKEN(INTEGER, -1619581104)
TOKEN(+)
TOKEN(INTEGER, -1619581072)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, -1619581008)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, )
TOKEN(,)
TOKEN(INTEGER, -1619580848)
TOKEN(,)
TOKEN(INTEGER, -1619580816)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 1900762352)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(-)
TOKEN(IDENTIFIKATOR)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(/)
TOKEN(IDENTIFIKATOR)
TOKEN(//)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, 1900762928)
TOKEN(>)
TOKEN(()
TOKEN(INTEGER, 1900762960)
TOKEN(+)
TOKEN(INTEGER, 1900762992)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, 1900763056)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, )
TOKEN(,)
TOKEN(INTEGER, 1900763216)
TOKEN(,)
TOKEN(INTEGER, 1900763248)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
1 -  chyba v programu v ramci lexikalni analyzy.

TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(INTEGER, 963097840)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN(,)
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(=)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(IDENTIFIKATOR)
TOKEN(-)
TOKEN(IDENTIFIKATOR)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(/)
TOKEN(IDENTIFIKATOR)
TOKEN(//)
TOKEN(IDENTIFIKATOR)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, 963098416)
TOKEN(>)
TOKEN(()
TOKEN(INTEGER, 963098448)
TOKEN(+)
TOKEN(INTEGER, 963098480)
TOKEN(*)
TOKEN(IDENTIFIKATOR)
TOKEN(+)
TOKEN(INTEGER, 963098544)
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(IDENTIFIKATOR)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN(STRING, )
TOKEN(,)
TOKEN(INTEGER, 963098704)
TOKEN(,)
TOKEN(INTEGER, 963098736)
TOKEN())
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(IDENTIFIKATOR)
TOKEN(()
